src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -579,54 +576,18 @@ public void setMinBitrate(int minBitrateBps)
     }
 
     /**
-     * Returns the number of milliseconds until this instance wants a worker
-     * thread to call {@link #run()}. The method is called on the same
-     * worker thread as Process will be called on.
-     *
-     * @return the number of milliseconds until this instance wants a worker
-     * thread to call {@link #run()}
-     */
-    @Override
-    public long getTimeUntilNextRun()
-    {
-        //@Todo In webrtc, this method is implemented as commented below.
-        //final long kDisabledModuleTime = 1000;
-        //return  kDisabledModuleTime;
-        //Clarify if we can leave it as implemented.
-        if (lastProcessTime < 0L)
-            return 0L;
-
-        synchronized (critSect)
-        {
-            return
-                lastProcessTime
-                    + processIntervalMs
-                    - System.currentTimeMillis();
-        }
-
-    }
-
-    /**
-     * When an object implementing interface <code>Runnable</code> is used
-     * to create a thread, starting the thread causes the object's
-     * <code>run</code> method to be called in that separately executing
-     * thread.
-     * <p>
-     * The general contract of the method <code>run</code> is that it may
-     * take any action whatsoever.
-     *
-     * @see Thread#run()
+     * {@code sendSideEstimate} is the wrapper for bitrate estimation when
+     * the bridge is the sender.
+     * @param ssrc is the SSRC associated with the RTP packet send by the bridge
+     * @param arrivalTimeMs is time the packet reached the client
+     * @param sendTimeMs is time the packet left the bridge
+     * @param packetSize is the size of the packet.
      */
     @Override
-    public void run()
+    public void sendSideEstimate(long ssrc, long arrivalTimeMs, long sendTimeMs, long packetSize)
```

Comment:
Why do we need to add a new method if it just calls incomingPacketInfo() with the arguments rearranged? For send-side bwe we know we will only be using RemoteBitrateEstimatorAbsSendTime so there's no need to pollute the RemoteBitrateEstimator with extra methods.